exercises/006-conversion/conversion.go

```diff
@@ -10,9 +11,9 @@ func main() {
 	var three int = 3
 	var oneThird float64 = 1.0 / 3.0
 	var twoPlusThree int = 2 + three
-	// These two variables are of distint type. To perform operations
+	// These two variables are of distinct types. To perform operations
 	// between variables they must be of the same type! Convert one of
-	// the variables on this line so that the result of 1/3+2+3 is printed out.
+	// the variables in the Println, line so that the result of 1/3+2+3 is printed out.
```

Comment:
The comma + "line" is off, maybe `// the variables in the Println so that the result of 1/3+2+3 is printed out.` is better?

Reply:
Yes it sounded a bit off thanks for the guidance!